sopt/src/main/scala/dagr/sopt/cmdline/ClpGroup.scala

```diff
@@ -21,18 +21,23 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
  */
-package dagr.core.cmdline
+
+package dagr.sopt.cmdline
 
 /**
- * Trait for groups of CommandLinePrograms.
- */
-trait PipelineGroup extends Ordered[PipelineGroup] {
+  * Trait for groups of CommandLinePrograms.
+  */
+trait ClpGroup extends Ordered[ClpGroup] {
```

Comment:
Could we still add `rank` here with a default value that's high enough that other classes can easily go above or below it, and then use that as the first order in `compare`?